packages/auth/package.json

```diff
@@ -114,6 +114,7 @@
     "@firebase/component": "0.6.4",
     "@firebase/logger": "0.4.0",
     "@firebase/util": "1.9.3",
+    "karma-cli": "2.0.0",
```

Comment:
Can you double check if this dependency is needed?

Reply:
nevermind